### DISCUSSION TITLE
Git Copilot Uses limit

### DISCUSSION THREAD

**Discussion by twt2shiv** [community/community#53956] [Copilot Conversations]
### Hi everyone,

I just want to know how many active logins I can use for git copilot in a different system with the same subscription account.

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @twt2shiv , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
